lib/index.js

```diff
@@ -94,7 +115,8 @@ api.add = (meta = {}, options = {}, callback) => {
           id: 'urn:uuid:' + results.generateUuid.uuid,
           ledger: options.ledgerId,
           ledgerNode: options.ledgerNodeId,
-          operationCollection: results.generateUuid.operationCollection
+          operationCollection: results.generateUuid.operationCollection,
+          services,
```

Comment:
Seems like this should be `plugins` and then each is namespaced by a name specified in the plugin.

@msporny, thoughts?

Reply:
I understand he first part, literally change `services` to `plugins`, but I'm not sure about the second part.  The plugin has a name which corresponds to what is passed to the `injector.use` API.  Are you suggesting that we record a string that is something other than the plugin name?

Reply:
Actually, I guess we already have what we need then. I was confusing how the injector works here with the other injectors (which may continue to be a problem but that's another issue entirely). I see that the plugin already has a name and separately an API type and API itself. So, I think we're good ... I just think we should expose the name in the calls for namespacing purposes. Thoughts?

cc: @msporny